packages/block-library/src/query/edit/inspector-controls/format-controls.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { SelectControl } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+
+// All WP post formats, sorted alphabetically by translated name.
+const POST_FORMATS = [
+	{ slug: 'aside', name: __( 'Aside' ) },
+	{ slug: 'audio', name: __( 'Audio' ) },
+	{ slug: 'chat', name: __( 'Chat' ) },
+	{ slug: 'gallery', name: __( 'Gallery' ) },
+	{ slug: 'image', name: __( 'Image' ) },
+	{ slug: 'link', name: __( 'Link' ) },
+	{ slug: 'quote', name: __( 'Quote' ) },
+	{ slug: 'standard', name: __( 'Standard' ) },
+	{ slug: 'status', name: __( 'Status' ) },
+	{ slug: 'video', name: __( 'Video' ) },
+].sort( ( a, b ) => {
+	const normalizedA = a.name.toUpperCase();
+	const normalizedB = b.name.toUpperCase();
+
+	if ( normalizedA < normalizedB ) {
+		return -1;
+	}
+	if ( normalizedA > normalizedB ) {
+		return 1;
+	}
+	return 0;
```

Comment:
I'm guessing this sorting is for localized labels.

Reply:
Yes